component/loki/source/kubernetes/kubetail/tailer.go

```diff
@@ -81,9 +81,9 @@ func newLabelSet(l labels.Labels) model.LabelSet {
 var retailBackoff = backoff.Config{
 	// Since our tailers have a maximum lifetime and are expected to regularly
 	// terminate to refresh their connection to the Kubernetes API, the minimum
-	// backoff starts at zero so there's minimum delay between expected
+	// backoff starts at one so there's minimum delay between expected
 	// terminations.
-	MinBackoff: 0,
+	MinBackoff: 1,
```

Comment:
The golang `time.Duration` value corresponds to nanosecond int64 values, so we should explicitly define the unit as well.

Also, after reading through the code could we please make this a bit smaller to avoid missing short-lived pods?

```suggestion
	MinBackoff: 10 * time.Millisecond,
```